src/sentry/static/sentry/app/views/onboarding/components/addInstallationInstructions.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
```

Comment:
Would you be able to put these in `onboarding/components/integration` just to keep a bit of a separation for the integration related things. Making it a bit easier for people who look at this code in the future, since this stuff is somewhat generically named.

Reply:
@EvanPurkhiser Would I be able to do it? Yes. Will I? ABSOLUTELY NOT. j/k, I think the change makes sense ;)